packages/aragon-api-react/src/index.js

```diff
@@ -122,27 +141,32 @@ function getAragonApiData(hookName) {
 const useAragonApi = () => ({
   ...getAragonApiData('useAragonApi()'),
   requestMenu,
+  requestPath,
   _sendMessage: postMessage,
 })
 
 // direct access hooks
 const useApi = () => getAragonApiData('useApi()').api
 const useAppState = () => getAragonApiData('useAppState()').appState
+const useApps = () => getAragonApiData('useApps()').apps
 const useConnectedAccount = () =>
   getAragonApiData('useConnectedAccount()').connectedAccount
-const useMenuButton = () => {
-  const { displayMenuButton, requestMenu } = getAragonApiData('useMenuButton()')
-  return [displayMenuButton, requestMenu]
-}
+const useMenuButton = () => [
```

Comment:
is this an API change as it returns an array tuple now?

Reply:
I don’t think there was any change here? It’s been returning an array since the [initial PR](https://github.com/aragon/aragon.js/pull/259/files#diff-df3ed8cb308cbe434aac1d4a00e05543R139).